src/migrateCalculatedMeasuresToActivePivot/removeCalculatedMemberDefinitionFromMDXAndGetCubeName.ts

```diff
@@ -0,0 +1,48 @@
+import { DataModel, CubeName, getCube } from "@activeviam/data-model";
+import {
+  MdxString,
+  MdxSelect,
+  getCalculatedMeasures,
+  getCubeName,
+  removeCalculatedMemberDefinition,
+  stringify,
+} from "@activeviam/mdx";
+import { parse } from "@activeviam/activeui-sdk";
+import _intersection from "lodash/intersection";
+
+export const removeCalculatedMemberDefinitionFromMDXAndGetCubeName = (
```

Comment:
The current name is pretty heavy.
I would make it simpler, and add a multiline comment to explain what it does.
```suggestion
export const migrateCalculatedMeasuresInMdx = (
```

Reply:
I agree the name was bad, this is definitely better.